File: API.Foodie/API.Foodie/Model/Dish.cs
namespace API.Foodie.Model;

public class Dish
{
    public int Id { get; set; }
    public string Name { get; set; }
    public TimeSpan CookingTime { get; set; }
    public string YouWillNeed { get; set; }
    public int DishWeight { get; set; }
    public decimal Price { get; set; }
    public bool IsVisible { get; set; }
    public string Ingredients { get; set; }

    public List<Photo> Photos { get; set; }
}

File: webClient.Foodie/src/app/models/dish.ts
import { Time } from "@angular/common";
import { Photo } from "./photo";

export interface Dish {
    id: number,
    name: string,
    cookingTime: Time,
    youWillNeed: string,
    dishWeight: number,
    price: number,
    isVisible: boolean,
    ingredients: string,
    photos: Photo[]
}
